Src/car.c

```diff
@@ -242,16 +240,18 @@ void taskBlink(void* can)
 		}
 		if(!HAL_GPIO_ReadPin(P_AIR_STATUS_GPIO_Port,P_AIR_STATUS_Pin))
 		{
+			HAL_GPIO_TogglePin(LD5_GPIO_Port, LD5_Pin);
 			tx.Data[0] |= 0b00001000;
 		}
-		if (xSemaphoreTake(car.m_CAN, 100) == pdTRUE)
-		{
+		//if (xSemaphoreTake(car.m_CAN, 100) == pdTRUE)
```

Comment:
Why is this commented?

Reply:
We don't use semaphores anymore. A queue is being used in it's place so there is no risk of it colliding on threads.

Reply:
We shouldn't be committing commented code. It's fine for now, we'll solve as part of #10. For future, this will not be allowed.